libs/partners/mongodb/langchain_mongodb/chat_message_histories.py

```diff
@@ -33,11 +39,17 @@ def __init__(
         session_id: str,
         database_name: str = DEFAULT_DBNAME,
         collection_name: str = DEFAULT_COLLECTION_NAME,
+        session_id_key: str = DEFAULT_SESSION_ID_KEY,
```

Comment:
Generally recommend using the `*` operator as much as possible to make things a bit more breaking changes proof during future refactors

```suggestion
       *,
        session_id_key: str = DEFAULT_SESSION_ID_KEY,
```